src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -5066,7 +5071,12 @@ private static string BuildSequenceOfBinaryExpressions_01(int count = 4096)
             return builder.ToString();
         }
 
-        [Fact, WorkItem(5395, "https://github.com/dotnet/roslyn/issues/5395")]
+#if TEST_IOPERATION_INTERFACE
```

Comment:
Rather than having an `#if` at every callsite why not define a special Fact that has hte conditional built in. 

``` 
class TestIOperationFact : Fact
#if !TEST_IOPERATION_INTERFACE
public string SkipReason => "Test ioperation interface"
#endif
```

Reply:
> Rather than having an #if at every callsite why not define a special Fact that has hte conditional built in.

I'll try that.